Code/GraphMol/FilterCatalog/Wrap/FilterCatalog.cpp

```diff
@@ -153,28 +153,28 @@ class PythonFilterMatch : public FilterMatcherBase {
     python::incref(functor);
   }
 
-  ~PythonFilterMatch() {
+  ~PythonFilterMatch() override {
     if (incref) python::decref(functor);
   }
-  virtual bool isValid() const {
+  bool isValid() const override {
```

Comment:
"In a member function declaration or definition, override ensures that the function is virtual and is overriding a virtual function from the base class."